ui/src/app/preferences/components/Routes/Routes.tsx

```diff
@@ -15,18 +15,30 @@ const Routes = (): JSX.Element => {
   return (
     <Switch>
       <Redirect exact from={prefsURLs.prefs} to={prefsURLs.details} />
-      <Route exact path={prefsURLs.details} render={Details} />
```

Comment:
These routes needed to be changed to render the component without passing props through the route.